fbpic/particles/deposition/cuda_methods.py

```diff
@@ -575,86 +572,81 @@ def deposit_chi_gpu_linear(x, y, z, w, q, M, inv_gamma,
         if i == 0:
             frequency_per_cell = np.int32(incl_offset)
 
-        for it in range(len(m_tuple)):
```

Comment:
Here I simply removed the loop over modes, since the kernel is now called once per mode.